resolwe_bio/processes/expression/salmon_index.yml

```diff
@@ -86,9 +117,12 @@
 
       salmon --no-version-check index \
         -t {{ nucl.fastagz.file }} \
+        {% if decoys %} -d {{ decoys.file.file }} {% endif %} \
+        {% if gencode %} --gencode {% endif %} \
+        {% if keep_duplicates %} --keepDuplicates {% endif %} \
+        {% if perfect_hash %} --perfectHash {% endif %} \
         -i salmon_index \
         -p {{ requirements.resources.cores }} \
-        --type quasi \
```

Comment:
Was this argument deprecated?

Reply:
The `quasi` is the only (and default) option for `--type` in recent version of Salmon, which removes the need to have this parameter exposed.